src/main/java/org/jetbrains/research/refactorinsight/data/RefactoringInfo.java

```diff
@@ -508,12 +502,14 @@ public RefactoringInfo setGroup(Group group) {
   public void correctLines(String before, String mid, String after) {
     boolean skipAnnotationsLeft = true;
     boolean skipAnnotationsRight = true;
-    if (name.matches("Add\\s(\\w)*\\sAnnotation")) {
-      skipAnnotationsRight = false;
-    } else if (name.matches("Remove\\s(\\w)*\\sAnnotation")) {
-      skipAnnotationsLeft = false;
-    } else if (name.matches("Modify\\s(\\w)*\\sAnnotation")) {
-      skipAnnotationsLeft = skipAnnotationsRight = false;
+    if (getName().endsWith("Annotation")) {
```

Comment:
Why should we use this check of the start and end of the string instead of regex?

Reply:
Fixed